addons/event_crm/models/event_lead_rule.py

```diff
@@ -81,11 +81,11 @@ class EventLeadRule(models.Model):
              'Per Order: A single Lead is created per Ticket Batch/Sale Order (B2B)')
     lead_creation_trigger = fields.Selection([
         ('create', 'Attendees are created'),
-        ('confirm', 'Attendees are confirmed'),
+        ('register', 'Attendees are registered'),
```

Comment:
Is it necessary to change the key ? Could lessen a bit the diff and upgrade script.

Reply:
Not really, I will keep the old one